provision/acc_provision/test_main.py

```diff
@@ -1771,10 +1938,18 @@ def run_provision(inpfile, expectedkube=None, expectedtar=None,
         copy_file(expectedapic, apicfile, args.debug, "/tmp/generated_apic.txt")
         copy_file(expectedtar, out_tar, args.debug, "/tmp/generated_operator.tar.gz")
 
-        compare_yaml(expectedkube, output, args.debug, "/tmp/generated_kube.yaml", cleanupFunc)
-        compare_yaml(expectedoperatorcr, operator_cr_output, args.debug, "/tmp/generated_operator_cr.yaml", cleanupFunc)
+        if args.compare_kube_yaml_aci_op_cm_in_plain_text:
+            result_kube_yaml = compare_kube_yaml(expectedkube, output, args.debug, "/tmp/generated_kube.yaml", cleanupFunc)
```

Comment:
do we need a result_kube_yaml ? , i think compare_kube_yaml itself can be used. And we can put the same name a compare_yaml

Reply:
its needed, compare_kube_yaml is not only comparing the kube yaml but also converting generated kube yaml's ac-operator-config configmap data from base64 to plaintext, so when fix-testdata.sh script is run that time copied generated kube yaml to testdata folder will have operator-config configmap data in plaintext